core/src/main/java/io/cucumber/core/options/CucumberPropertiesParser.java

```diff
@@ -77,7 +79,7 @@ public RuntimeOptionsBuilder parse(Map<String, String> properties) {
 
         parse(properties,
             FILTER_TAGS_PROPERTY_NAME,
-            Function.identity(),
+            TagExpressionParser::parse,
```

Comment:
We'll have to look at this.

Reply:
Currently the problem is that it doesn't report back where the problem came from.